api/src/components/authorization/controller.ts

```diff
@@ -72,11 +72,27 @@ export const authorize = async (event: I.APIRequest<I.AuthorizeRequestBody>): Pr
             })
         );
 
+        // eslint-disable-next-line @typescript-eslint/dot-notation
```

Comment:
This is either a interface mismatch with how it's accessing or another problem. Temp disabled this ts error.